### ISSUE TITLE
 New updated List of User Guide Issues

### ISSUE THREAD

**Issue by swzCuroverse** [common-workflow-language/user_guide#253]
Making a list of existing user guides issues to help with the upcoming Outreachy project and well as help sort out what is left after @kinow recent update work. 

**v1.1, v1.2 features**

- [ mention version mixing; and explain when and how to use cwl-upgrader ]   https://github.com/common-workflow-language/user_guide/issues/188
- [NetworkAccess: Need example to show how to Indicate whether a process requires outgoing IPv4/IPv6 network access and what happens if not indicated and network access is need but isn't available]  https://github.com/common-workflow-language/user_guide/issues/249

**formating**

- [Tabe of Contents doesn't show important workflow subsections ] https://github.com/common-workflow-language/user_guide/issues/252 
-

**Comment by swzCuroverse** [comment]
I am working on this - but if you are heading to this ticket from Outreachy - all the above is up to date, it is just not complete.  I should have it finished by October 11th 2022.  I am trying to also add new tickets as they come to make it a bit of a "living" ticket to help us keep track of where in the User Guide these new issues fit in.

**Comment by Smyja** [comment]
> I am working on this - but if you are heading to this ticket from Outreachy - all the above is up to date, it is just not complete. I should have it finished by October 11th 2022. I am trying to also add new tickets as they come to make it a bit of a "living" ticket to help us keep track of where in the User Guide these new issues fit in.

Hey, do you mean you are already resolving every issue or you mean you are still updating the list of issues.

**Comment by swzCuroverse** [comment]
> 

No, I  not doing these. This is just meant to be a issue of all issues that need to be done. I am just taking ownership of moving all the known to-do to this ticket. I wouldn't mind help on that if someone wanted to help on that as well. I will change my comment above to make it clearer.

**Comment by Smyja** [comment]
> > 
> 
> No, I not doing these. This is just meant to be a issue of all issues that need to be done. I am just taking ownership of moving all the known to-do to this ticket. I wouldn't mind help on that if someone wanted to help on that as well. I will change my comment above to make it clearer.

Okay. Some issues aren't listed here, like the one I raised and was merged. Also the pygments/code highlighting issue isn't here. Should I leave it? I have already started working on it though.

**Comment by Fienne** [comment]
> 

@swzCuroverse , the issues about fixing missing links under the command line isn't really clear to me. I don't understand where to find and replace the correct links.
Can you please help me out?

**Comment by swzCuroverse** [comment]
@Fienne I would skip those for now.  I need to get more information about that.  There is something that @kinow indicated in the read-me about an issue with graphs.  I will dig into that a bit more - probably easier for you to look into one of the issues that has a ticket already written.

**Comment by AfiMaameDufie** [comment]
Hi @swzCuroverse ,which issue can I pick up to work on?

**Comment by Emenyi95** [comment]
Hello @swzCuroverse . My name is Emenyi Nelly and I am an Outreachy intern for the current session. Please can an issue be assigned to me to work on, as I am very much interested in documentation.

**Comment by pauline-banye** [comment]
Hi @swzCuroverse I'm Pauline, an outreachy intern. I'm excited to work on this project. Where do I start?

**Comment by swzCuroverse** [comment]
Hi all - please pick any ticket you find interesting above or in the general list of tickets.  You ca join us on the Matrix channel here if you want guidance in setting up.  https://app.element.io/#/room/#docs-training:matrix.org  See the contribution guide to get started .

**Comment by chrilona** [comment]
Hello everyone,Christabel Lona Atieno just joining to learn and grow from this interaction.

**Comment by DBAKITA** [comment]
Hello @swzCuroverse in reference to the command line tool topic

 Topics/CommandLineTool

 Spaces in commands , https://github.com/common-workflow-language/user_guide/issues/39
 Arguments (tell the reader the different use cases for arguments and inputs, tell them there is a section about inputs), No existing ticket just a common bottom of the markdown

Here is my input:
**How do commandlines work**
A Command Line Tool is a non-interactive executable program that reads some input, performs a computation, and terminates after producing some output. Command line programs are a flexible unit of code sharing and reuse, unfortunately the syntax and input/output semantics among command line programs is extremely heterogeneous. A common layer for describing the syntax and semantics of programs can reduce this incidental complexity by providing a consistent way to connect programs together. This specification defines the Common Workflow Language (CWL) Command Line Tool Description, a vendor-neutral standard for describing the syntax and input/output semantics of command line programs.

**Different use cases for arguments and inputs**
Available primitive types are string, int, long, float, double, and null; complex types are array and record; in addition there are special types File, Directory and Any.

The following example demonstrates some input parameters with different types and appearing on the command line in different ways.

**First, create a file called inp.cwl, containing the following:**

inp.cwl
#!/usr/bin/env cwl-runner

cwlVersion: v1.0
class: CommandLineTool
baseCommand: echo
inputs:
  example_flag:
    type: boolean
    inputBinding:
      position: 1
      prefix: -f
  example_string:
    type: string
    inputBinding:
      position: 3
      prefix: --example-string
  example_int:
    type: int
    inputBinding:
      position: 2
      prefix: -i
      separate: false
  example_file:
    type: File?
    inputBinding:
      prefix: --file=
      separate: false
      position: 4

outputs: []

Sometimes tools require additional command line options that don’t correspond exactly to input parameters.

In this example, we will wrap the Java compiler to compile a java source file to a class file. By default, “javac” will create the class files in the same directory as the source file. However, CWL input files (and the directories in which they appear) may be read-only, so we need to instruct “javac” to write the class file to the designated output directory instead.

arguments.cwl
#!/usr/bin/env cwl-runner

cwlVersion: v1.0
class: CommandLineTool
label: Example trivial wrapper for Java 9 compiler
hints:
  DockerRequirement:
    dockerPull: openjdk:9.0.1-11-slim
baseCommand: javac
arguments: ["-d", $(runtime.outdir)]
inputs:
  src:
    type: File
    inputBinding:
      position: 1
outputs:
  classfile:
    type: File
    outputBinding:
      glob: "*.class"
